packages/applications-service-api/src/controllers/documents.v3.js

```diff
@@ -41,7 +47,59 @@ const buildFilters = (req) => ({
 	datePublishedTo: req.body.datePublishedTo
 });
 
+const documentTypeDict = {
+	'RULE 6 LETTER': {
+		bo: 'Rule 6 letter',
+		ni: 'Rule 6 letter - Notification of the preliminary meeting and matters to be discussed'
+	},
+	'RULE 8 LETTER': {
+		bo: 'Rule 8 letter',
+		ni: 'Rule 8 letter - notification of timetable for the examination'
+	},
+	'EXAMINATION LIBRARY': {
+		bo: 'Examination library',
```

Comment:
i think in back office the value is `Exam library` from what i've seen in the database. Might be worth checking in their repo, api spec or message schema

Reply:
where would this be stored ? 

i can not find a ref to it.

Reply:
on back office this gets set when editing the metadata for a document - 

![Screenshot 2023-08-10 at 15 49 29](https://github.com/Planning-Inspectorate/applications-service/assets/33483916/1bed2e1b-a8d6-4ca2-b0e8-388871eb437d)


possible values are defined here:
https://github.com/Planning-Inspectorate/back-office/blob/7399139990d1f745d190bef012d8977f0506e7bd/apps/web/src/server/applications/case/documentation-metadata/documentation-metadata.controller.js#L78-L99

so looks to me like its `Exam library`

and also those "decision letter" documents _are_ using the same naming as NI (albeit with slightly different casing)

Reply:
fixed exam lib